string objectName
string limbSide 
string gripperOrientation 
---
int64 success_bool
